cmd/tether/ops_linux.go

```diff
@@ -177,33 +178,85 @@ func (t *operations) SetupFirewall(config *tether.ExecutorConfig) error {
 			ifaceName := iface.Attrs().Name
 			log.Debugf("slot %d -> %s", endpoint.ID, ifaceName)
 
-			established.Interface = ifaceName
-			_ = established.Commit(context.TODO())
-
-			// handle the ports
-			for _, p := range endpoint.Ports {
-				// parse the port maps
-				r, err := portToRule(p)
-				if err != nil {
-					log.Errorf("can't apply port rule (%s): %s", p, err.Error())
-					continue
+			switch endpoint.Network.TrustLevel {
+			case common.Open:
+				// Accept all incoming and outgoing traffic
+				generalPolicy(netfilter.Accept)
+
+			case common.Closed:
+				// Reject all incoming and outgoing traffic
+				generalPolicy(netfilter.Drop)
+
+			case common.Outbound:
+				// Reject all incoming traffic, but allow outgoing
+				setupOutboundFirewall(ifaceName)
+
+			case common.Peers:
+				// Outbound + all ports open to source addresses in --container-network-ip-range
+				setupOutboundFirewall(ifaceName)
```

Comment:
I was just thinking that it might be useful to make API calls to an external service or something similar. Actually, having published ports might be useful too, in certain circumstances.